src/jdk.compiler/share/classes/com/sun/tools/javac/tree/TreeCopier.java

```diff
@@ -491,6 +491,13 @@ public JCTree visitInstanceOf(InstanceOfTree node, P p) {
         return M.at(t.pos).TypeTest(expr, pattern);
     }
 
+    @DefinedBy(Api.COMPILER_TREE)
+    public JCTree visitAnyPattern(AnyPatternTree node, P p) {
+        JCBindingPattern t = (JCBindingPattern) node;
```

Comment:
not sure this is correct, we are returning a JCBindingPattern?

Reply:
+1